jumpscale/clients/explorer/workloads.py

```diff
@@ -240,3 +241,52 @@ def sign_delete(self, workload_id: int, tid: int, signature: bytes) -> bool:
         data = j.data.serializers.json.dumps({"signature": signature, "tid": tid, "epoch": j.data.time.now().timestamp})
         self._session.post(url, data=data)
         return True
+
+    def list_by_gevent(
```

Comment:
gevent is merely an implementation detail, the name should get changed to list_workloads maybe

Reply:
sure :D